builds/e2e/templates/e2e-setup.yaml

```diff
@@ -139,7 +139,7 @@ steps:
   - pwsh: |
       $manifestSignerClientDir = '$(Build.SourcesDirectory)/samples/dotnet/ManifestSignerClient'
       dotnet build $manifestSignerClientDir
-      $manifestSignerClientBinDir = Convert-Path "$manifestSignerClientDir/bin/Debug/netcoreapp3.1"
+      $manifestSignerClientBinDir = Convert-Path "$manifestSignerClientDir/bin/Debug/net6.0"
```

Comment:
You will need to update e2e-setup-base-image-update-release.yaml as well?

Reply:
Yep I think so, good catch. Also compare-compatability.yaml